workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
 set -e
+oc adm wait-for-stable-cluster --minimum-stable-period=1m --timeout=5m
```

Comment:
5m might be too short. for example, if we want to run kube-burner after an upgrade, I am seeing some of the pods (kubeapi) take > 5 min just to roll out.

Reply:
The control plane pods are supposed to be rolled out once the upgrade is finished, another potential issue is the mcp rollout, which runs asynchronously of the `oc adm upgrade` command.

Also, I think we're not running any workload after an upgrade completion

Reply:
I have tried to set `--minimum-stable-period` to 1m and I saw kube-burner errors due to the cluster still stabilizing, but keeping it at 5min I have had better results, just my current experience...